localstack/constants.py

```diff
@@ -184,3 +179,43 @@
 
 # user that starts the elasticsearch process if the current user is root
 OS_USER_ELASTICSEARCH = "localstack"
+
+# Thundra related constants
+########################################################################################################################
+
+
+def get_latest_version(metadata_url):
```

Comment:
Thanks for adding this functionality @serkan-ozal .

We usually use `constants.py` for literally only string constants, and we wouldn't want to make an HTTP call each time this file gets loaded on startup. Can we load the latest version info on demand, when required in the context of a Lambda invocation?

Also, I'm thinking that it would make sense to encapsulate this code into a separate `localstack/plugins/thundra.py` file. (Later (once we have the Lambda executor plugin mechanism in place), we can encapsulate the entire Thundra-related logic in that separate file...)

Reply:
@whummer :+1: Handled at https://github.com/localstack/localstack/commit/b0340f36f5634cd7c261b07fa036981b3c637a02